### ISSUE TITLE
Highlight "top libraries"

### ISSUE THREAD

**Issue by markerikson** [markerikson/redux-ecosystem-links#60]
Since a lot of the categories have many libraries listed, it would be helpful to highlight the top 2-4 "must use" libraries in each category, if appropriate.

I should figure out how to format this, and which ones I want to highlight.

**Comment by alexkrolick** [comment]
How about a ⭐️ ?

**Comment by markerikson** [comment]
Probably a good idea.  I don't normally use emojis anywhere, but that ought to work.

**Comment by jrnail23** [comment]
This would be really helpful.  There are just so many libs listed, and a lot of them appear to be either dead projects or almost never used, according to their npm stats.

**Comment by markerikson** [comment]
@jrnail23 : yeah.  What I'd _really_ like, tbh, is a tool that parses the Markdown files and generates an interactive list that could show Github stars and update info, NPM stats, etc.  There's already existing examples of tools and sites along those lines, I just don't have time to work on anything like that myself.

**Comment by shth** [comment]
@markerikson Can you share those examples of tools?

**Comment by markerikson** [comment]
- Someone specifically wrote a tool to parse my React/Redux links list, and turn it into a REST API : https://github.com/satiewaltz/reactionary .  This addons list uses effectively identical Markdown formatting, so I would expect that the tool would work if it was pointed at this list.
- There's a sorta-similar but much smaller Redux addons list at https://devarchy.com/redux . The source is at https://github.com/devarchy/website , and I once asked a question about it at https://github.com/devarchy/website/issues/3 .
- The "Awesome React" page at LibHunt.com is very similar to what I'd like done for my list: https://react.libhunt.com/ .  I actually emailed them a few weeks ago asking if they had any interest in using this list as a source for Redux stuff, but never heard back.
- List.community is another similar "nice UI over awesome lists" site: https://list.community/ .  Looks like the source is at https://github.com/listcommunity/support , and according to https://www.producthunt.com/posts/list-community it's built with CRA, and uses a free Algolia index from their "Github Autocomplete" project ( https://github.com/algolia/github-awesome-autocomplete ).
- Yarn's package search page is also neat and vaguely relevant: https://yarnpkg.com/en/packages , https://blog.algolia.com/yarn-search-javascript-packages/ , https://github.com/yarnpkg/website/tree/master/js/src

So yeah, I don't have time to build a "search the libs list" app myself, but it would be incredibly cool if someone else were to tackle it :)  It seems like all the pieces you'd need are available already.  Parse the links list, grab Github URLs and package names, grab NPM metadata and Github stats, mash 'em up, and provide a UI that shows items based on their category and allows sorting and filtering.

**Comment by markerikson** [comment]
I threw together a quick UI mockup to illustrate the kind of thing that I'm picturing:

![addons-list-ui-mockup.png](https://i.imgur.com/Eh6wSpX.png)

**Comment by wilbertliu** [comment]
Hey all,

After some discussions with @markerikson, I decided to work on this with Dave (I don't know his handle on GitHub yet). We will update you soon when there's something that we can show.

Wish us luck.. 😀🙏🏻

**Comment by markerikson** [comment]
And for reference, looks like the new repo and initial design discussion is at https://github.com/wilbertliu/redux-ecosystem/issues/1 .

**Comment by jondot** [comment]
@markerikson I've tweaked [Celeste](https://github.com/jondot/celeste) to work with this repo's format, here's the result:

https://gist.github.com/jondot/10c2ddefe6b411a1f3049005fce716df

It also found a bunch of 404s while scoring.

Let me know if this is something you like, and i'll work to integrate it into this repo for all files.

**Comment by markerikson** [comment]
Huh.  That is very intriguing.

https://github.com/wilbertliu/redux-ecosystem would still be a neat thing to get working, but looks like @wilbertliu stopped working on it a year ago.

Sure, if you can figure out how to get things set up to improve this repo, that'd be great!

Wonder if it would be useful for https://github.com/markerikson/react-redux-links in some way too.
